combat/CombatSystem.cpp

```diff
@@ -1746,6 +1747,31 @@ namespace {
                                launches_event);
 
                 DebugLogger(combat) << "Attacker: " << attacker->Name();
+
+                // Set launching carrier as at least basically visible to other empires.
+                if (launches_event) {
```

Comment:
does this test do anything?

Reply:
It should have tested that there were actually launch events